### PR TITLE
Organizing infra repo

### PR DESCRIPTION
**Pull request by akristen**
this PR reorganizes the infrastructure section of the docs repo.

- removes random subdirectories
- combines docs into easier to locate subdirectories
- aligns the repo with the yaml file
- updates urls to cleanly mirror docs titles and left nav
- removes a few docs
- renames i18n files

to do:

investigate why some i18n files were deleted

### CODE REVIEW

File: src/nav/infrastructure.yml [hunk 1]
```diff
@@ -2,134 +2,133 @@ title: Infrastructure monitoring
...
-            path: /docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos
+            path: /docs/infrastructure/infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos
       - title: Windows installation
         pages:
           - title: MSI install (preferred)
-            path: /docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows
+            path: /docs/infrastructure/infrastructure-agent/windows-installation/windows-msi-installer
           - title: Zip assisted install
-            path: /docs/infrastructure/install-infrastructure-agent/windows-installation/zip-assisted-install-infrastructure-agent-windows
+            path: /docs/infrastructure/infrastructure-agent/windows-installation/windows-zip-assisteds
```
Comment (paperclypse) [inline src/nav/infrastructure.yml:49]: @akristen this link led to a 404

File: src/nav/infrastructure.yml
Comment (paperclypse) [inline src/nav/infrastructure.yml:1]: @akristen This is broken https://deploy-preview-19146--docs-website-netlify.netlify.app/install/elasticsearch

Comment (paperclypse) [reply src/nav/infrastructure.yml:1]: And this one too: https://deploy-preview-19146--docs-website-netlify.netlify.app/docs/infrastructure/new-relic-infrastructure/troubleshooting/agent-not-starting-there-are-no-logs

**Review by paperclypse — COMMENTED** [review]
@akristen I'll approve this once you fix the small number of broken links I found

File: src/nav/infrastructure.yml
Comment (akristen) [reply src/nav/infrastructure.yml:1]: the elasticsearch one i think is some persisting issue that i've seen in other PRs. i didnt touch any install docs... i can see what the problem is tho

Comment (paperclypse) [reply src/nav/infrastructure.yml:1]: Yeah, I noticed that you hadn't touched that one. Might as well fix it while you're there, right? :) Thanks!

**Review by paperclypse — APPROVED** [review]
Looks good, thanks!

**Pull request merged**